src/helper/getAuthPageTemplate.ts

```diff
@@ -47,12 +56,34 @@ export function getAuthPageTemplate(
           style="
             font-family: sans-serif;
             font-size: smaller;
-            color: #85888d;
+            color:#FFFFFF;
           "
         >
           ${info}
         </p>
-      </div>
+        <p style="
+            font-family: sans-serif;
+            font-size: smaller;
+            color:#FFFFFF;
+            class="countdown" id="countdown">
+            Window will be closed in 5 second(s)
+          </p>
+        </div>
+
+        <script>
+          let seconds = ${closeTime};
+          const countdownElement = document.getElementById('countdown');
+
+          const countdown = setInterval(function() {
+            countdownElement.textContent = '${closeMessage}'.replace('{seconds}', seconds) + ' second(s)';
+            seconds--;
+
+            if (seconds < 1) {
+              clearInterval(countdown);
+              window.close();
+            }
```

Comment:
Checking endTime before does make a difference in UX. if its after it gets closed right at 1. but checking it before will allow us to close window after completion of that last 1s, run with this and you will experience this. 
```suggestion
       
            if (seconds < 1) {
              clearInterval(countdown);
              window.close();
            }
            
            countdownElement.textContent = '${closeMessage}'.replace('{seconds}', seconds) + ' second(s)';
            seconds--;
```

lemme know once this is done @Spiral-Memory , we can merge this.

Reply:
done